contracts/instruments/RibbonThetaVaultYearn.sol

```diff
@@ -715,9 +726,11 @@ contract RibbonThetaVaultYearn is DSMath, OptionsVaultStorage {
         returns (uint256)
     {
         uint256 total =
-            wmul(lockedAmount, collateralToken.pricePerShare()).add(
-                assetBalance()
-            );
+            wmul(
+                lockedAmount,
+                collateralToken.pricePerShare().mul(_decimalShift())
+            )
+                .add(assetBalance());
```

Comment:
shift by decimals everywhere is only change. 

in case of weth decimal shift will be 10 ** 0